### DISCUSSION TITLE
Where do I set ESLINT_USE_FLAT_CONFIG=false?

### DISCUSSION THREAD

**Discussion by staraphd** [eslint/eslint#18915] [Q&A]
I want to use eslint v9 with old .eslintrc config file. 
Where do I need to write `ESLINT_USE_FLAT_CONFIG=false` in the files mentioned below and if and what other changes do I need to make to use eslint v9?
CI invokes lint using `npm run lint`

**package.json**
```
  "scripts": {
    "lint": "eslint . --ext .ts,.spec.ts",
}
```

**.eslintrc (in root folder)**
```
{
    "root": true,
    "parser": "@typescript-eslint/parser",
    "plugins": [
      "@typescript-eslint"
    ],
    "extends": [
      "eslint:recommended",
      "plugin:@typescript-eslint/eslint-recommended",
      "plugin:@typescript-eslint/recommended"
    ],
    "rules": { }
  }
```

**Comment by nzakas** [comment]
See https://www.npmjs.com/package/cross-env
